ee/docker-ee/ubuntu.md

```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
 [Get Docker Engine - Community for Ubuntu](/install/linux/docker-ce/ubuntu.md)
 **instead of this topic**.
```

Comment:
In my opinion, this line should be the first line of content on the page (following the Docker Enterprise blurb, that is). If possible, it should also render as Important (if that is an option..not quite a Warning, but stronger than just a Note).

Also, the line that follows should be edited to read: 

"To get started with Docker Engine - Enterprise on Ubuntu, confirm that the prerequisites are met and then install Docker."